nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ExecuteSQL.java

```diff
@@ -278,8 +289,18 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
                 } else {
```

Comment:
Above this line is a call to `session.remove`. Do you see any reason to note add an "original" relationship to the processor so the input flowfile is not lost?

Reply:
Thanks for the review. As for adding an "original" relationship, that's outside the scope of this ticket. This is a bug fix ticket, with no intention of adding new functionality, just fixing a bug (that I introduced in a previous ticket...)

Reply:
Ok.